src/nvim/if_cscope.c

```diff
@@ -1454,7 +1477,7 @@ static char *cs_manage_matches(char **matches, char **contexts, int totmatches,
  *
  * parse cscope output
  */
-static char *cs_parse_results(int cnumber, char *buf, int bufsize, char **context, char **linenumber, char **search)
+static char *cs_parse_results(size_t cnumber, char *buf, int bufsize, char **context, char **linenumber, char **search)
```

Comment:
Could you break this line please? Thankfully, lines of this length are becoming a rarity.